Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.fedoraproject.org/fedora:41
+
+LABEL maintainer "Akashdeep Dhar <t0xic0der@fedoraproject.org>"
+
+ENV PYTHONUNBUFFERED=1
+
+RUN dnf -y install python3-pip
+RUN dnf -y clean all
+RUN pip3 install --upgrade firmitas==0.1.3a0
+
+ENV FIRMITAS_CONFIG=/etc/firmitas/conf/myconfig.py
+
+ENTRYPOINT firmitas --conffile $FIRMITAS_CONFIG
```

Comment:
Best to add ENTRYPOINT and CMD like so https://github.com/fedora-infra/mote/blob/main/Dockerfile#L26-L27

Keep the app/binary in ENTRYPOINT, and any params being passed in CMD.

Reply:
Roger. Adding those.